<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StadiumTripPage</class>
 <widget class="QDialog" name="StadiumTripPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="StadiumText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stadium Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Next_Button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Next Stadium</string>
   </property>
  </widget>
  <widget class="QLabel" name="WelcomeText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>631</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome to the home of the [Team Name Here]!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DistanceText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Distance Traveled: 0 Miles</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="SouvenirComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SouvenirQuantity">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>48</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPurchase">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchase</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total: $</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotal">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableShop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableCart">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>